sys-cluster/genders/genders-1.27.3.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+DISTUTILS_USE_SETUPTOOLS=no
+
+inherit autotools distutils-r1 perl-module java-pkg-opt-2
+
+case "${PV}" in
+9999)
+	SRC_URI=""
+	EGIT_REPO_URI="https://github.com/chaos/${PN}.git"
```

Comment:
You could just plaintext the `PN` since it's not a changing variable, and makes copypasting easier.